src/EventSubscriber/FixedWindowRateLimiterSubscriber.php

```diff
@@ -0,0 +1,90 @@
+<?php
+
+/**
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; under version 2
+ * of the License (non-upgradable).
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ *
+ * Copyright (c) 2021 (original work) Open Assessment Technologies SA;
+ */
+
+declare(strict_types=1);
+
+namespace OAT\SimpleRoster\EventSubscriber;
+
+use Psr\Log\LoggerInterface;
+use Symfony\Component\EventDispatcher\EventSubscriberInterface;
+use Symfony\Component\HttpKernel\Event\RequestEvent;
+use Symfony\Component\HttpKernel\Exception\TooManyRequestsHttpException;
+use Symfony\Component\HttpKernel\KernelEvents;
+use Symfony\Component\RateLimiter\RateLimiterFactory;
+
+class FixedWindowRateLimiterSubscriber implements EventSubscriberInterface
```

Comment:
Let's make this feature togglable by introducing a `IS_RATE_LIMITER_ENABLED` environment variable with default value `false`.

Reply:
Actually, if you do not specify any route on RATE_LIMITER_FIXED_WINDOW_ROUTES, by default the feature will bbe disabled. Do you see the necessity to add a new env var?

Reply:
IMO only `RATE_LIMITER_FIXED_WINDOW_ROUTES` is enough.